src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -120,8 +120,8 @@ class GroupTagValues extends AsyncComponent<
             payload={{
               queryType: ExportQueryType.IssuesByTag,
               queryInfo: {
-                project_id: group.project.id,
-                group_id: group.id,
+                project: group.project.id,
+                group: group.id,
```

Comment:
These fields for `Issues-by-Tag` exports were changed up a bit so the `project` field matches the one in the discover PR